src/utils/general.ts

```diff
@@ -176,7 +176,7 @@ export function getQueryForDepositsAndRewards(accountid: string) {
 
 export function convertFromUtezToTez(amountInUtez: number) {
   const tezAmount = amountInUtez / 1000000;
```

Comment:
This must use metadata, take a look at the implementation of number formatting in Arronax

Reply:
Used the `Intl.NumberFormat` based on number formatting in `Arronax`